resources/views/backend/books/edit.blade.php

```diff
@@ -31,7 +31,7 @@
                     <div class="row">
                       <div class="col-xs-12">
                         <label for="exampleInputTitle">{{ __('books.title') }}</label>
-                        <input type="text" class="form-control" name="title" id="exampleInputTitle" placeholder="{{ __('books.Title') }}" value="{!! old('title') !!}">
+                        <input type="text" class="form-control" name="title" id="exampleInputTitle" placeholder="{{ __('books.title') }}" value="{{ $book->title }}">
```

Comment:
A xin lỗi mọi người, bữa trước a review ko kỹ, remove cái chữ `example` trong tất cả mấy cái `for` và `id` của thẻ `label` và `input` nha.
Vì app này mình làm là thật, không phải là demo, nên không để `example`.